about.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Lato" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/about.css">
+    <title>About Kadoma Community Development</title>
+</head>
+<body>
+    <nav>
+        <ul class="d-lg-flex d-none justify-content-end sp-bg list-unstyled">
+            <li class="px-1 py-1">
+                <a href="#"> <img class="cont" src="img/fb.png" alt="facebook-icon"></a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#"> <img class="cont" src="img/twitter.png" alt="twitter-icon"></a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">Logout</a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">My Page </a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">English</a>
+            </li>
```

Comment:
Provide icons instead images this is the design requested:
![Screen Shot 2021-09-15 at 16 37 07](https://user-images.githubusercontent.com/22258274/133513075-cc90ec4c-f39e-463e-a844-055a34fdd50e.png)

**Yours:**
![Screen Shot 2021-09-15 at 16 35 11](https://user-images.githubusercontent.com/22258274/133513111-6e41e3a0-a827-471f-b9ca-f1eed5c95000.png)

#
**Kindly fix this:**
- Your logo image is overlapping the upper bar:
![Screen Shot 2021-09-15 at 16 38 58](https://user-images.githubusercontent.com/22258274/133513261-9a6deb08-6bf0-4751-9cd6-3573d165bd96.png)
##